cnf-certification-test/platform/suite.go

```diff
@@ -114,8 +118,30 @@ var _ = ginkgo.Describe(common.PlatformAlterationTestKey, func() {
 			ginkgo.Skip(" non ocp cluster ")
 		}
 	})
+
+	testID = identifiers.XformToGinkgoItIdentifier(identifiers.TestServiceMeshIdentifier)
+	ginkgo.It(testID, ginkgo.Label(testID), func() {
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.DebugPods)
+		TestServiceMesh(&env)
+	})
 })
 
+func TestServiceMesh(env *provider.TestEnvironment) {
+	var badPods []string
+	for _, put := range env.Pods {
+		data := put.Containers
+		fmt.Println(data)
+		for _, d := range data {
+			if d.Status.Name == istio {
+				tnf.ClaimFilePrintf("For pods %s ,ns %s we have service mesh", put.Data.Name, put.Data.Namespace)
+				continue
```

Comment:
why u are doing continue and not break here? is it not enough that one of the containers has istio?